### PR TITLE
Port `OpenButton` component to 0.5 syntax

### PR DESCRIPTION
**Pull request by euclio**
#### Summary

Updates the `OpenButton` component to 0.5 syntax and adds the "open_button" example back.

cc #100.

#### Checklist

- [x] cargo fmt
- [x] cargo clippy
- [x] cargo test
- [x] updated CHANGES.md

### CODE REVIEW

File: relm4-components/src/open_button/factory.rs [hunk 1]
```diff
@@ -11,53 +11,33 @@ pub(crate) struct FileListItem {
...
-        let key = key.clone();
-        label.connect_clicked(move |_| {
-            send!(sender, OpenButtonMsg::OpenRecent(key.clone()));
-        });
-        FileListItemWidgets { label, row }
+    view! {
+        gtk::ListBoxRow {
+            gtk::Button {
+                set_label: self.path.iter().last().expect("Empty path").to_str().unwrap(),
+                inline_css: "margin: 0;",
```
Comment (AaronErhardt) [inline relm4-components/src/open_button/factory.rs:28]: This should rather use `set_margin_all` from WidgetPlus.

**Review by AaronErhardt — CHANGES_REQUESTED** [review]
Looks good to me! Just two minor things should be fixed.

File: examples/relm4-components/.recent_files [hunk 2]
```diff
@@ -0,0 +1 @@
+/home/euclio/down/650180847.pdf
```
[the remark this answers is not in the document]
Comment (euclio) [reply examples/relm4-components/.recent_files]: Whoops! Fixed, and added a .gitignore.

File: relm4-components/src/open_button/factory.rs (hunk 1 above)
Comment (euclio) [reply relm4-components/src/open_button/factory.rs]: Fixed.

**Review by AaronErhardt — APPROVED** [review]
Thanks!

**Pull request merged**